src/web/experiments/tests/sign-in-gate-mandatory.ts

```diff
@@ -7,8 +7,8 @@ export const signInGateMandoryTest: ABTest = {
 	author: 'Peter Colley',
 	description:
 		'Compare mandatory gate (an article sigin gate without the dismiss button) with the main signin gate',
-	audience: 0.2,
-	audienceOffset: 0.7,
+	audience: 0.3,
+	audienceOffset: 0.6,
```

Comment:
just curious if it is intended that both these numbers do not reach 1?

Reply:
it's intended, the main variant uses 0.0-0.6, the mandatory experiment uses 0.6-0.9 and the long running control uses 0.9-1.0